crates/l2/starknet-proxy-client/src/deploy.rs

```diff
@@ -29,9 +29,12 @@ pub enum Error {
     DeployContract(#[from] ethereum_instance::Error),
 }
 
-const UNSAFE_PROXY: &str = include_str!("artifacts/UnsafeProxy.json");
-const SAFE_PROXY_3_0_2: &str = include_str!("artifacts/Proxy_3_0_2.json");
-const SAFE_PROXY_5_0_0: &str = include_str!("artifacts/Proxy_5_0_0.json");
+const UNSAFE_PROXY: &str = include_str!("../../../../artifacts/UnsafeProxy.json");
+// TODO : find the contract in repos
```

Comment:
what does this TOOD man? also, do you think we should just delete unsafe proxy one, i don't know if its being used now. will madara use it?

Reply:
I added this here by mistake.